integration_tests/test_split_request_params.py

```diff
@@ -0,0 +1,85 @@
+import pytest
```

Comment:
can you please run, black/isort/ruff here? 

Why tf is precommit ci not working 😭 . (I'll have a look) https://pre-commit.ci/